documents/motions/02.2018.1 - Definitions.md

```diff
@@ -32,7 +32,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Committee</td>
-      <td>A committee of the WCA that has been appointed by the WCA under the terms of the WCA Bylaws or any of the Motions. Leaders of committees are WCA Staff and rest of committees members are WCA External Staff.</td>
+      <td>A Committee of the WCA that has been appointed by the WCA under the terms of the WCA Bylaws or any of the Motions. Leaders and Senior Members of Committees are WCA Staff and rest of Committee members are WCA External Staff.</td>
```

Comment:
The definition of "Committee" uses the word "Committee"?

Reply:
I think this is okay. It's not so much the definition of committee that is important, as much as to make it clear that when we use the word committee, this is what we are referring to.